feature_layers/feature-collection-layer-from-portal/src/main/java/com/esri/samples/feature_collection_layer_from_portal/FeatureCollectionLayerFromPortalSample.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Esri.
```

Comment:
Don't forget to update the copyright year 😉 

```suggestion
 * Copyright 2020 Esri.
```